docs/provenance.md

```diff
@@ -52,6 +52,12 @@ $ helm package --sign --key 'helm signing key' --keyring path/to/keyring.secret
 **TIP:** for GnuPG users, your secret keyring is in `~/.gnupg/secring.gpg`. You can
 use `gpg --list-secret-keys` to list the keys you have.
 
+**Warning:**  the latest GnuPG version store your secret keyring using a new format 'kbx' on the default location  '~/.gnupg/pubring.kbx'. Please use the following command to convert your keyring to the legacy gpg format:
```

Comment:
Hey! Can we make this say "GnuPG v2" just to make it clear what version we're talking about?

Reply:
I update read me to refers to "GnuPG v2" instead of 'latest GnuGP version"